README.md

```diff
@@ -137,6 +137,7 @@ or for installing in [development mode](https://pip.pypa.io/en/latest/cli/pip_in
 
 
 ```sh
+python -m pip install -r requirements-dev.txt
 python -m pip install -ve . --no-build-isolation --config-settings=editable-verbose=true
```

Comment:
If you don't need an editable installation, you can leave out the ``--no-build-isolation`` part
(which will then let pip pull dependencies for you)

I'm thinking that we should really remove the whole development mode section from the README, though.
You probably don't want to do this unless you want to work on pandas itself, and I think the docs here are much better at explaining things.
https://pandas.pydata.org/docs/development/contributing_codebase.html